src/Caching/SqlServer/src/DatabaseOperations.cs

```diff
@@ -26,6 +26,8 @@ internal sealed class DatabaseOperations : IDatabaseOperations
     /// </summary>
     private const int DuplicateKeyErrorId = 2627;
 
+    private const string UtcNowParameterKey = "UtcNow";
```

Comment:
```suggestion
    private const string UtcNowParameterName = "UtcNow";
```

I'm not specifically familiar with this code, but the parameter receiving this as an argument is called "name", rather than "key".

Reply:
It wasn't causing any issue @amcasey, it is just for hygiene like you mentioned 👍🏿 

Also updated the variable name to `UtcNowParameterName` like you suggested.